lib/srv/discovery/common/watcher.go

```diff
@@ -41,6 +41,8 @@ type WatcherConfig struct {
 	FetchersFn func() []Fetcher
 	// Interval is the interval between fetches.
 	Interval time.Duration
+	// PollTrigger can be used to force an instant Poll, instead of waiting for the next poll Interval.
+	PollTrigger chan struct{}
```

Comment:
Poll Trigger is quite enigmatic. Can we rename it to something like: 
```suggestion
	TriggerFetchC chan struct{}
```

Or DiscoveryConfigChangedC

Reply:
I like `TriggerFetchC`, thank you.